src/main.rs

```diff
@@ -296,9 +296,14 @@ fn get_metadata(messages: &[Message]) -> CTRConfig {
         _ => artifact.target.name,
     };
 
+    let author = match std::env::var("CARGO_PKG_AUTHORS") {
+        Ok(name) => name,
+        Err(_) => String::from("Unspecified Author"), // as standard with the devkitPRO toolchain
+    };
```

Comment:
I would rather use the data we get from `cargo_metadata`. Why can't we do the following?
```suggestion
    let author = match &package.authors {
        [name, ..] => name,
        [] => String::from("Unspecified Author"), // as standard with the devkitPRO toolchain
    };
```

Reply:
I don't see any reason for the authors field to go away (it's still a valid Cargo.toml field, and isn't changing schema). Also, the RFC doesn't say to use the env variable instead of cargo_metadata, just that it's possible to use the env variable.

Reply:
As I discussed with @ian-h-chamberlain : yes, all you have said is completely true. If the majority wants it the other way I’ll change it no prob.